Update/omot_012.txt

```diff
@@ -1653,7 +1653,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	if (GetGlobalFlag(GADVMode)) { OutputLine(NULL, "", NULL, "<size=-2>", Line_Continue); }
 	OutputLine(NULL, "一体誰が考えたんだ、この罰ゲームは？！　クラスの男女交際事情を考えてみても、他の奴が引いたら確実にスルーされるセーフカードじゃないか！　明らかに俺と魅音を狙い撃ちしたとしか思えんぞ！！",
-			NULL, "Who on earth thought up this punishment?! Just considering the gender relationships in the class, isn't this an unsafe card even if someone else were to draw it?! And apparently everyone has their sights set on Mion and me!!", Line_Normal);
+			NULL, "Who on earth thought up this punishment?! Just considering the gender relationships in the class, this is a safe card for everyone else because they'd surely disregard it even if they were to draw it! It was obviously created to set on Mion and me!!", Line_Normal);
```

Comment:
```suggestion
			NULL, "Who on earth thought up this punishment?! Just considering the gender relationships in the class, this is a safe card for everyone else because they'd definitely disregard it even if they drew it! It was obviously created specifically for Mion and me!!", Line_Normal);
```